maxim/logger/langchain/utils.py

```
@@ -588,7 +604,11 @@ def parse_langchain_messages(
                         )
                     elif message_type.endswith("ToolMessage"):
                         messages.append(
-                            {"role": "tool", "content": message.content or ""}
+                            {
+                                "role": "tool",
+                                "content": message.content or "",
+                                "tool_call_id": message.tool_call_id,
+                            }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)